src/mesh/mesh_io.jl

```diff
@@ -196,6 +196,11 @@ function load_mesh_serial(mesh_file::AbstractString; n_cells_max, RealT)
     end
 
     mesh = CurvedMesh(size, mapping; RealT=RealT, unsaved_changes=false, mapping_as_string=mapping_as_string)
+  elseif mesh_type == "UnstructuredQuadMesh"
+    mesh_filename = h5open(mesh_file, "r") do file
+      return read(attributes(file)["mesh_filename"])
+    end
+    mesh = UnstructuredQuadMesh(mesh_filename; RealT=RealT, periodicity=false, unsaved_changes=false)
```

Comment:
Would it make sense to store periodicity in the mesh file? Right now, you hardcode `periodicity=false` here, which would preclude this from working with periodic mesh files, I guess.

Reply:
Since this is only for plotting the periodicity can be set to false. The periodicity is important in order to compute the coupling correctly during the simulation but does not influence the visualization. (I just tested this again to double check)

Reply:
Discussed in https://github.com/trixi-framework/Trixi.jl/pull/575#discussion_r631600499 ff

Reply:
Done. I added a FIXME